feat(cci): add cciv2_pod and cciv2_deployment container field (#6855)
huaweicloud/terraform-provider-huaweicloud: docs/incubating/cciv2_deployment.md

  The [resources](#containers_resources) structure is documented below.

* `security_context` - (Optional, List) Specifies the security context of the container.
  The [security_context](#containers_security_context) structure is documented below.

* `startup_probe` - (Optional, List) Specifies the startup probe of the container.
  The [startup_probe](#containers_probe) structure is documented below.

* `stdin` - (Optional, Bool) Specifies whether this container should allocate a buffer for stdin in the container runtime.

* `stdin_once` - (Optional, Bool) Specifies whether this container runtime should close the stdin channel.

* `termination_message_path` - (Optional, String) Specifies the termination message path of the CCI pod container.

* `termination_message_policy` - (Optional, String) Specifies the termination message policy of the CCI pod container.

* `tty` - (Optional, Bool) Specifies whether this container should allocate a TTY for itself.

* `working_dir` - (Optional, String) Specifies the working directory of the CCI Pod container.

* `volume_mounts` - (Optional, List) Specifies the volume mounts probe of the container.

>   The [startup_probe](#containers_volume_mounts) structure is documented below.

Reviewer: ```suggestion
  The [volume_mounts](#containers_volume_mounts) structure is documented below.
```

<a name="containers_volume_mounts"></a>

> The `env` block supports:

Reviewer: ```suggestion
The `volume_mounts` block supports:
```

> * `extend_path_mode` - (sub_path, String) Specifies the name of the volume mounts.

Reviewer: ```suggestion
* `extend_path_mode` - (Optional, String) Specifies the extend path mode of the volume mounts.
```

  The [secret_ref](#containers_env_from_secret_ref) structure is documented below.

<a name="containers_env_from_config_map_ref"></a>
The `config_map_ref` block supports:

* `name` - (Optional, String) Specifies the name.

* `optional` - (Optional, Bool) Specifies whether to be defined.

<a name="containers_env_from_secret_ref"></a>
The `secret_ref` block supports:

* `name` - (Optional, String) Specifies the name.

* `optional` - (Optional, Bool) Specifies whether to be defined.

<a name="containers_lifecycle"></a>
The `lifecycle` block supports:

* `post_start` - (Optional, List) Specifies the lifecycle post start of the CCI pod container.
  The [post_start](#containers_lifecycle_post_start) structure is documented below.

* `pre_stop` - (Optional, List) Specifies the lifecycle pre stop of the CCI pod container.
  The [pre_stop](#containers_lifecycle_pre_stop) structure is documented below.

<a name="containers_lifecycle_post_start"></a>
The `post_start` block supports:

* `exec` - (Optional, List) Specifies the lifecycle post start of the CCI pod container.
  The [exec](#containers_lifecycle_post_start_exec) structure is documented below.

> * `http_get` - (Optional, List) Specifies the lifecycle pre stop of the CCI pod container.

Reviewer: ```suggestion
* `http_get` - (Optional, List) Specifies the lifecycle post start of the CCI pod container.
```

  The [http_get](#containers_lifecycle_post_start_http_get) structure is documented below.

<a name="containers_lifecycle_post_start_exec"></a>

> The `http_get` block supports:

Reviewer: ```suggestion
The `exec` block supports:
```

* `command` - (Optional, List) Specifies the command line to execute inside the container.

<a name="containers_lifecycle_post_start_http_get"></a>
The `http_get` block supports:

* `host` - (Optional, String) Specifies the host name.

* `http_headers` - (Optional, List) Specifies the custom headers to set in the request.
  The [http_headers](#containers_lifecycle_post_start_http_get_http_headers) structure is documented below.

* `path` - (Optional, String) Specifies the path to access on the HTTP server.

* `port` - (Optional, String) Specifies the port to access on the HTTP server.

* `scheme` - (Optional, String) Specifies the scheme to use for connecting to the host.

<a name="containers_lifecycle_post_start_http_get_http_headers"></a>
The `http_headers` block supports:

* `name` - (Optional, String) Specifies the name of the custom HTTP headers.

* `value` - (Optional, String) Specifies the value of the custom HTTP headers.

<a name="containers_lifecycle_pre_stop"></a>
The `pre_stop` block supports:

> * `exec` - (Optional, List) Specifies the lifecycle post start of the CCI Pod container.

Reviewer: ```suggestion
* `exec` - (Optional, List) Specifies the lifecycle pre stop of the CCI Pod container.
```

  The [exec](#containers_lifecycle_pre_stop_exec) structure is documented below.

* `http_get` - (Optional, List) Specifies the lifecycle pre stop of the CCI Pod container.
  The [http_get](#containers_lifecycle_pre_stop_http_get) structure is documented below.

<a name="containers_lifecycle_pre_stop_exec"></a>

> The `http_get` block supports:

Reviewer: ```suggestion
The `exec` block supports:
```

  The [http_get](#containers_probe_http_get) structure is documented below.

* `initial_delay_seconds` - (Optional, Int) Specifies the number of seconds after the container has started
  before liveness probes are initialed.

* `period_seconds` - (Optional, Int) Specifies how often to perform the probe.

* `termination_grace_period_seconds` - (Optional, Int) Specifies the termination grace period seconds.

<a name="containers_probe_exec"></a>
The `exec` block supports:

* `command` - (Optional, List) Specifies the command line to execute inside the container.

<a name="containers_probe_http_get"></a>
The `http_get` block supports:

* `host` - (Optional, String) Specifies the host name.

* `http_headers` - (Optional, List) Specifies the custom headers to set in the request.
  The [http_headers](#containers_probe_http_get_http_headers) structure is documented below.

* `path` - (Optional, String) Specifies the path to access on the HTTP server.

* `port` - (Optional, String) Specifies the port to access on the HTTP server.

* `scheme` - (Optional, String) Specifies the scheme to use for connecting to the host.

<a name="containers_probe_http_get_http_headers"></a>

> The `scheme` block supports:

Reviewer: ```suggestion
The `http_headers` block supports:
```

* `name` - (Optional, String) Specifies the name of the custom HTTP headers.

* `value` - (Optional, String) Specifies the value of the custom HTTP headers.

<a name="containers_ports"></a>
The `ports` block supports:

* `container_port` - (Required, Int) Specifies the number of port to expose on the IP address of pod.

* `name` - (Optional, String) Specifies the port name of the container.

* `protocol` - (Optional, String) Specifies the protocol for container port.

<a name="containers_resources"></a>
The `resources` block supports:

* `limits` - (Optional, Map) Specifies the limits of resource.

* `requests` - (Optional, Map) Specifies the requests of the resource.

<a name="containers_security_context"></a>
The `security_context` block supports:

* `capabilities` - (Optional, List) Specifies the capabilities of the security context.
  The [capabilities](#containers_security_context_capabilities) structure is documented below.

> * `proc_mount` - (Optional, String) Specifies the denotes the type of proc mount to use for the containers.

Reviewer: the description maybe wrong
